bedrock/utils/views.py

```diff
@@ -0,0 +1,33 @@
+from django.views.generic import TemplateView
+
+from lib.l10n_utils import LangFilesMixin
+
+
+class VariationMixin(object):
+    template_name_variations = None
+    template_context_variations = None
+
+    def get_template_names(self):
+        names = super(VariationMixin, self).get_template_names()
+        if self.template_name_variations:
```

Comment:
I haven't yet seen an instance where an experiment used fewer custom templates than variations, but I don't think this is a bad scenario for which to prepare. The downside is the values for `template_name_variations` and `template_context_variations` will often be the same/repeated, but that's not a big sacrifice and shouldn't be too error prone.